notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -2467,6 +2467,58 @@ computer hacking [Digicorp tried to "jack in" to the Sunway Systems computer sys
+
+:: Ratings
+4 <paul>
+
+:: Major Themes
+extraterrestrial being [The story follows Graham and his family as they barricade themselves in their isolated farmhouse in the midst of an alien invasion of Earth.]
+crackpot theory [The film confronts the viewer with the idea that crop circles are being made by extraterrestrials.]
+family affairs [The story explored how the close knit Hess family came together in the wake of Colleen Hess dying tragically in a traffic accident. Her husband Graham lost his faith, their two young children struggled to understand why she had died, and Graham's younger brother Merrill moved to their farm to be there for them.]
+father and son [Graham and his relationship with his young son Morgan was explored throughout the story.]
+father and daughter [Graham and his relationship with his young daughter Bo was explored throughout the story.]
+brother and brother [Graham and his younger brother Merrill were coordinating to protect Graham's young kids (and themselves) from nefarious space aliens.]
+uncle and niece [Merrill had moved to the farm to help care for his young niece, Bo.]
+uncle and nephew [Merrill had moved to the farm to help care for his young nephew, Morgan.]
+brother and sister [Morgan and his kid sister Bo were caught up in all the goings on surrounding the alien invasions, and the apparent arrival of aliens on their farm.]
+coping with a crisis of faith [The story explores how Graham, a former Episcopal priest, lost his faith after his wife died tragically in a traffic accident.]
+coping with the death of a spouse [Graham was having a hard time accepting that his wife was really gone.]
+attack from outer space [The story shows an alien invasion unfolding from the vantage point of an isolated family farm.]
+
+:: Minor Themes
+human health condition [Morgan suffered from asthma and had a bad attack on night.]
+coping with personal failure [The former minor league baseball player Merrill was evidently a bit torn up inside over never having made it to the majors.]
+remorse [Ray expressed regret for having caused Colleen's death and Graham's subsequent losing of his faith.]
+law enforcement [The local police officer Caroline Paski visited the Hess house a couple of times to investigate the strange goings on there.]
+coping with the death of a parent [The Hess children were coming to terms with that their mother had died and was never coming back.]
+hoaxes in society [A man on television stated that the crop circles had been dismissed as hoaxes in the 1980s.]
+speculative spaceship [The television news showed alien spacecrafts in the nigh sky above Mexico City.]
+the game of baseball [A local man showered Merrill with praise for for having hit a record breaking 507 foot home run in the minor leagues.]
+nuclear weapons [Morgan read in his UFO book that invading aliens would use ground tactics and hand-to-hand combat to prevent humans from nuking Earth, making the planet useless to them.]
```

Comment:
That made zero sense to me.

Reply:
(perhaps it was meant not to?)

Reply:
I simplified the motivation, but the whole scenario as described in the film made no sense at all, lol.